packages/plugin-core/src/test/suite-integ/LookupProviderV3.test.ts

```diff
@@ -336,4 +338,71 @@ suite("onAccept", () => {
       });
     }
   );
+  describeMultiWS(
+    "GIVEN a note with invalid name that already exists",
+    {
+      preSetupHook: async ({ wsRoot, vaults }) => {
+        await NoteTestUtilsV4.createNote({
+          vault: vaults[0],
+          wsRoot,
+          fname: "foo. bar.baz",
+          body: "note with invalid name",
+          genRandomId: true,
+        });
+      },
+      timeout: 5e3,
+    },
+    () => {
+      test("THEN accept lookup", async () => {
+        await VSCodeUtils.closeAllEditors();
+        const cmd = new NoteLookupCommand();
+        const { provider } = await cmd.gatherInputs();
+        const note = (
+          await ExtensionProvider.getEngine().findNotes({
+            fname: "foo. bar.baz",
+          })
+        )[0];
+        const item = {
+          ...note,
+          label: "foo. bar.baz",
+          detail: "",
+          alwaysShow: true,
+        };
+        const out = provider.shouldRejectItem!({ item });
+        expect(out).toBeFalsy();
+        cmd.cleanUp();
+      });
+    }
+  );
+  describeMultiWS(
+    "GIVEN a new item with invalid name",
+    {
+      preSetupHook: ENGINE_HOOKS.setupBasic,
+      timeout: 5e3,
```

Comment:
was the timeout necessary here? just checking that these tests were actually taking 5s

Reply:
Got into a habit of adding timeouts in test to give time for the CI to run it when other tests were blocking it, but not really necessary here. I'll remove